src/packages/v2v3/components/Create/hooks/useLoadInitialStateFromQuery.ts

```diff
@@ -101,7 +101,7 @@ const parseCreateFlowStateFromInitialState = (
   }
 
   return {
-    projectChainId: 0, // not necessary for v2v3
+    projectChainId: 84532, // not necessary for v2v3
```

Comment:
Is this necessary? Also this is in `v2v3` directory. Is it used in V4? If it is, can we move it somewhere else?